go-controller/README.md

```diff
@@ -0,0 +1,83 @@
+# ovn kubernetes go-controller
+
+The golang based ovn controller is a reliable way to deploy the OVN SDN using kubernetes clients and watchers based on golang. 
+
+### Build
+
+Ensure go version >= 1.8
```

Comment:
There was a reason, but now I cannot recall it.
Will fix if I cannot recollect soon.